testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -7017,24 +7025,26 @@ void checkRegexpExtract(SqlOperatorFixture f0, FunctionAlias functionAlias) {
     final Consumer<SqlOperatorFixture> consumer = f -> {
       f.checkScalarApprox("log(10, 10)", "DOUBLE",
           isWithin(1.0, 0.000001));
-      f.checkScalarApprox("log(64, 8)", "DOUBLE",
+      f.checkScalarApprox("log(8, 64)", "DOUBLE",
           isWithin(2.0, 0.000001));
-      f.checkScalarApprox("log(27,3)", "DOUBLE",
+      f.checkScalarApprox("log(3,27)", "DOUBLE",
           isWithin(3.0, 0.000001));
-      f.checkScalarApprox("log(100, 10)", "DOUBLE",
-          isWithin(2.0, 0.000001));
       f.checkScalarApprox("log(10, 100)", "DOUBLE",
+          isWithin(2.0, 0.000001));
+      f.checkScalarApprox("log(100, 10)", "DOUBLE",
           isWithin(0.5, 0.000001));
-      f.checkScalarApprox("log(cast(10e6 as double), 10)", "DOUBLE",
+      f.checkScalarApprox("log(10, cast(10e6 as double))", "DOUBLE",
```

Comment:
I think these tests would be easier to read if you wrote 1e7.

Reply:
done